decidim-core/config/initializers/rack_attack.rb

```diff
@@ -14,6 +14,11 @@
     &:ip
   )
 
+  Rack::Attack.blocklist("block all access to system") do |request|
+    # Requests are blocked if the return value is truthy
+    request.path.start_with?("/system") unless Decidim.system_whitelist_ips.map { |ip_address| IPAddr.new(ip_address).include?(IPAddr.new(request.ip)) }.any?
```

Comment:
Maybe we can rewrite it in a more performant (less iterations) and easier way to understand?
```suggestion
    if request.path.start_with?("/system")
      Decidim.system_whitelist_ips.any? && Decidim.system_whitelist_ips.map { |ip_address| IPAddr.new(ip_address).include?(IPAddr.new(request.ip)) }.none?
    end
```